app/src/main/java/com/yjooooo/sopt28th/ui/signin/SignInActivity.kt

```diff
@@ -0,0 +1,87 @@
+package com.yjooooo.sopt28th.ui.signin
+
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import androidx.activity.result.ActivityResult
+import androidx.activity.result.contract.ActivityResultContracts
+import com.yjooooo.sopt28th.R
+import com.yjooooo.sopt28th.databinding.ActivitySignInBinding
+import com.yjooooo.sopt28th.ui.base.BindingActivity
+import com.yjooooo.sopt28th.ui.home.HomeActivity
+import com.yjooooo.sopt28th.ui.signup.SignUpActivity
+import com.yjooooo.sopt28th.util.StatusBarUtil
+import com.yjooooo.sopt28th.util.ToastMessageUtil
+
+class SignInActivity : BindingActivity<ActivitySignInBinding>(R.layout.activity_sign_in) {
+    private val signUpActivityLauncher = registerForActivityResult(
+        ActivityResultContracts.StartActivityForResult()
+    ) {
+        setIdPwAfterSignUp(it)
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        Log.d("LifeCycle", "SignIn_onCreate")
+        StatusBarUtil.setStatusBar(this, resources.getColor(R.color.main_color_purple, null))
+        setOnLoginBtnClick()
+        setOnSignUpBtnClick()
+    }
+
+    override fun onStart() {
+        super.onStart()
+        Log.d("LifeCycle", "SignIn_onStart")
+    }
+
+    override fun onResume() {
+        super.onResume()
+        Log.d("LifeCycle", "SignIn_onResume")
+    }
+
+    override fun onPause() {
+        super.onPause()
+        Log.d("LifeCycle", "SignIn_onPause")
+    }
+
+    override fun onStop() {
+        super.onStop()
+        Log.d("LifeCycle", "SignIn_onStop")
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        Log.d("LifeCycle", "SignIn_onDestroy")
+    }
+
+    private fun setOnLoginBtnClick() {
+        binding.signInBtnLogin.setOnClickListener {
+            if (isUserInfoNotBlank()) {
+                ToastMessageUtil(this, binding.signInEdtId.text.toString() + "님 로그인되었습니다.")
```

Comment:
```suggestion
fun Context.ToastMessageUtil(message: String) {
    Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
}
ToastMessageUtil(binding.signInEdtId.text.toString() + "님 로그인되었습니다.")
```

이런 식으로 만들면 매개변수를 하나 더 줄일 수 있지 않을까?